build-tools-internal/version.properties

```diff
@@ -1,5 +1,5 @@
 elasticsearch     = 9.0.0
-lucene            = 9.11.1
+lucene            = 9.12.0-snapshot-a7ce3466d7c
```

Comment:
This should be a release version? We can't have a snapshot in main since it will go to serverless

Reply:
I see you mentioned this will wait for the release, I'm just pointing it out so there is no chance it slips in. :)